src/lib/utils.ts

```diff
@@ -1,6 +1,142 @@
-import { type ClassValue, clsx } from "clsx"
-import { twMerge } from "tailwind-merge"
+import { database } from "@/app/firebaseConfig";
+import { type ClassValue, clsx } from "clsx";
+import {
+  DataSnapshot,
+  limitToFirst,
+  onValue,
+  orderByChild,
+  orderByKey,
+  query,
+  ref,
+  startAt,
+} from "firebase/database";
+import { Dispatch, SetStateAction } from "react";
+import { twMerge } from "tailwind-merge";
 
 export function cn(...inputs: ClassValue[]) {
```

Comment:
@CristhianRF24 what means the nema method 'cn'. Is a reserved word in other case please rename